### PR TITLE
voice creation using a builder class and bug fixes

### PR DESCRIPTION
**Pull request by olajhidey**
Added some few changes, like creating calls with a builder class and some bug fixes.
also removed the org.json dependency from `pom.xml` and few general clean ups

### CODE REVIEW

File: src/main/java/io/github/signalwirecommunity/App.java [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
+                .machineDetection(MachineDetection.Enable)
+                .record(false)
+                .build();
+
+        System.out.println(voice.getResponse());
+
+        Call call = client.voice().update("57466289-00c2-4e27-a9ef-d42ebe385fa9", voice.getResponse());
+//
+        System.out.println(call.getAccount_sid());
+
```
Comment (danieleds) [inline src/main/java/io/github/signalwirecommunity/App.java]: could you remove personal information such as tokens, project ids, space names, phone numbers, etc?

Comment (olajhidey) [reply src/main/java/io/github/signalwirecommunity/App.java]: totally missed that. fixed in the latest commit

**Review by danieleds — APPROVED** [review]

**Comment by danieleds** [comment]
cool! Will you also push an update to maven?

**Comment by olajhidey** [comment]
> cool! Will you also push an update to maven?

Yeah done that, it might take quite a while before it reflects. looking to use github actions to automate the process so on merger to main the deployment kicks in

**Pull request merged**